src/components/avatar-audio-source.js

```diff
@@ -100,11 +100,11 @@ AFRAME.registerComponent("avatar-audio-source", {
     APP.dialog.on("stream_updated", this._onStreamUpdated, this);
     this.createAudio();
 
-    let audioOutputModePref = APP.store.state.preferences.audioOutputMode;
+    let disableLeftRightPanningPref = APP.store.state.preferences.disableLeftRightPanning === true;
```

Comment:
I think you can drop `=== true`.

Reply:
In this case (same as in media-video) I think we want to only recreate in case it went from undefined/false <-> true and not if it changed from undefined <-> false

Reply:
Thinking about it again now that we don't have the chat command, I don't think there is a case where this goes from `undefined` to `false` so I think is safe to remove that comparison.